.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: Attach Build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: macos-latest
```

Comment:
Does  it use Xcode 12? This has some implications and forces developers to use latest Xcode too. 

We should keep a fixed version (Xcode 11.7), so we can mention in the docs.

Reply:
Latest uses macos-10.15 which defaults to XCode 11.7. However,  I think its better to explicitly keep it as `macos-10.15` instead of `macos-latest`.

Reply:
yes, better keep track of used versions